util/configure

```diff
@@ -172,6 +173,9 @@ for my $opt (@ARGV) {
             $prefix = '.';
         }
 
+    } elsif ($opt =~ /^--ngx_prefix=(.*)/) {
```

Comment:
`--ngx-prefix` should be used instead of `--ngx_prefix` in the command line options.